common/src/main/java/org/apache/auron/conf/AuronConf.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.auron.conf;
+
+import org.apache.auron.jni.AuronAdaptor;
+
+@SuppressWarnings("unused")
+public enum AuronConf {
+    /// suggested batch size for arrow batches.
+    BATCH_SIZE("spark.auron.batchSize", 10000),
```

Comment:
`spark.` prefix is not needed, we can derive a `AuronSparkConf` in auron-spark and add the prefix there.

Reply:
good catch